client/src/routes/Sessions/components/SessionModal.tsx

```diff
@@ -58,11 +58,25 @@ const SessionModal = () => {
     useSessionNotificationReminder(session);
 
   const startTime = dayjs(session.startTime);
-
+  const startingNow = dayjs().isAfter(startTime.subtract(10, 'minutes'));
```

Comment:
I would make this into a function, dayjs() is like Date.now but now will be when the component renders not when the button is actually pressed. We only need to know now when the button is actually pressed so no need to do a dyajs() call until needed. So, do `cont startingNow = () => dayjs()...` and then `onPress` do `startingNow() ? ...`